### PR TITLE
chore: re-enable end-to-end tests

### PR DESCRIPTION
**Pull request by fdlk**
#### Checklist
- [ ] Functionality works & meets specifications
- [ ] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] User documentation updated
- [ ] Clean commits
- [ ] No warnings during install

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -85,7 +85,7 @@ pipeline {
             }
             container('rancher') {
                 sh "rancher apps delete ${NAME} || true" 
-                sh "sleep 5s" // wait for deletion
+                sh "sleep 15s" // wait for deletion
```
Comment (jelmerveen) [inline Jenkinsfile:88]: why tripple the amount?

File: Jenkinsfile [hunk 2]
```diff
@@ -26,7 +26,7 @@ pipeline {
           }
         }
         container('node') {
-          sh "daemon --name=sauceconnect -- /usr/local/bin/sc -u ${SAUCE_CRED_USR} -k ${SAUCE_CRED_PSW} -i ${TUNNEL_IDENTIFIER}"
+          startSauceConnect()
```
Comment (jelmerveen) [inline Jenkinsfile:29]: jenkins script?

File: Jenkinsfile (hunk 1 above)
Comment (fdlk) [reply Jenkinsfile:88]: the deploy failed with a 409 error in one of the builds, same happened in frontend repo where this fixed it

File: Jenkinsfile (hunk 2 above)
Comment (fdlk) [reply Jenkinsfile:29]: yes, we've added the magic formula to https://github.com/molgenis/molgenis-jenkins-pipeline/ in case we need to update it again

**Review by jelmerveen — APPROVED** [review]

**Review by jelmerveen — APPROVED** [review]

**Pull request merged**